### PR TITLE
Feature/davis346

### PR DESCRIPTION
**Pull request by eliasm**

### CODE REVIEW

**Comment by eliasm** [comment]
@baumlin please take a look.

File: davis_ros_driver/src/driver.cpp [hunk 1]
```diff
@@ -126,23 +133,40 @@ void DavisRosDriver::caerConnect()
...
-           davis_info_.logicVersion);
+          davis_info_.deviceID, davis_info_.deviceIsMaster, davis_info_.dvsSizeX, davis_info_.dvsSizeY,
+          davis_info_.logicVersion);
 
   // Send the default configuration before using the device.
   // No configuration is sent automatically!
   caerDeviceSendDefaultConfig(davis_handle_);
+  /*
+   * Something with the default aps size is wrong with the DAVIS346B. Quickfix with sending hardcoded values
+   */
```
Comment (baumlin) [inline davis_ros_driver/src/driver.cpp]: this hack is still necessary.. we will let you know when this changes.

File: davis_ros_driver/src/driver.cpp [hunk 2]
```diff
@@ -258,26 +283,93 @@ void DavisRosDriver::changeDvsParameters()
...
-        caerDeviceConfigSet(davis_handle_, DAVIS_CONFIG_BIAS, DAVIS240_CONFIG_BIAS_DIFFBN,
-            caerBiasCoarseFineGenerate(CF_N_TYPE(current_config_.DiffBn_coarse, current_config_.DiffBn_fine)));
-        caerDeviceConfigSet(davis_handle_, DAVIS_CONFIG_BIAS, DAVIS240_CONFIG_BIAS_ONBN,
-            caerBiasCoarseFineGenerate(CF_N_TYPE(current_config_.ONBn_coarse, current_config_.ONBn_fine)));
-        caerDeviceConfigSet(davis_handle_, DAVIS_CONFIG_BIAS, DAVIS240_CONFIG_BIAS_OFFBN,
-            caerBiasCoarseFineGenerate(CF_N_TYPE(current_config_.OFFBn_coarse, current_config_.OFFBn_fine)));
-
-        caerDeviceConfigSet(davis_handle_, DAVIS_CONFIG_BIAS, DAVIS240_CONFIG_BIAS_REFRBP,
-            caerBiasCoarseFineGenerate(CF_P_TYPE(current_config_.RefrBp_coarse, current_config_.RefrBp_fine)));
+        // VDAC
```
Comment (baumlin) [inline davis_ros_driver/src/driver.cpp]: those are still the best bias settings we have for the davis346b. (almost no rebound spikes, but comes with a tradeoff of a slight imbalance between ON and OFF events)

**Comment by eliasm** [comment]
thanks, @baumlin

**Comment by epeefencer** [comment]
Why am I getting this spam???

**Pull request merged**